docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -14,13 +14,13 @@ ms.assetid: 615db1ec-6ef3-4de2-bae6-4586affa9771
 
 # Getting started with F# with command-line tools
 
-This article covers how you can get started with using F# on .NET Core with the ..NET Core SDK 1.0 Preview 2.  It will go through building a multi-project solution with a Class Library, a Console App, and an xUnit test project.
+This article covers how you can get started with using F# on .NET Core 1.0 with the .NET Core SDK 2.0.1 Preview 2.  It will go through building a multi-project solution with a Class Library that is called by a Console Application.
 
 ## Prerequisites
 
-To begin, you must install the [.NET Core SDK 1.0 Preview 2](https://www.microsoft.com/net/core).
+To begin, you must install the [.NET Core SDK 2.0.1-preview2](dot.net/core).  If you have installed a previous version, you must uninstall that previous version.
```

Comment:
I don't think the link will work correctly without `https://`. At least on GitHub, this currently links to https://github.com/dotnet/core-docs/blob/update-fs-cli/docs/fsharp/tutorials/getting-started/dot.net/core (treating the link as relative).